website_portal_comment_attachment/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Portal Comment Attachment",
+    "summary": "Add an attachment on any portal comments",
+    "version": "12.0.1.0.0",
+    "category": "Website",
+    "website": "http://github.com/OCA/website",
+    "author": "Osiell, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "website",
```

Comment:
Is website dep needed? Wouldn't `portal` be enough?

Reply:
I confirm `portal` is enough. I would change this consequently

Reply:
true